src/screens/CardInfoScreen.tsx

```diff
@@ -1,20 +1,112 @@
 import React from 'react';
 import { StyleSheet, Text, View, StatusBar } from 'react-native';
 
-export default function CardInfoScreen() {
-  return (
-    <View style={styles.container}>
-      <Text>This is card info screen screen</Text>
-      <StatusBar barStyle="default" />
-    </View>
-  );
+import TopBar from '../components/TopBar';
+import TapBar from '../components/TapBar';
+import CardTile from '../components/CardTile';
+import Tile from '../components/Tile';
+import BenefitTile from '../components/BenefitTile';
+import CustomButton from '../components/CustomButton';
+import BoxContainer from '../components/BoxContainer';
+
+const SHOW_VIRTUAL = 1;
+
+export default function CardInfoScreen({ navigation, Card }) {
+  /*
+  todo:
+  check if virtual or real - if real barcode if virtual as implemented now
+  style components to match figma prototype
+  ^status as of rn: chaos
+  proper card icon
+  proper account balance from api
+  proper descriptions
+  proper everything (after api implementation)
+  */
+
+  if (SHOW_VIRTUAL /* */) {
```

Comment:
Nie podoba mi się sposób w jaki chcesz to renderować.
Można to zgrabniej załatwić:
```
return (
SHOW_VIRTUAL && (renderowany tsx dla tego warunku)
!SHOW_VIRTUAL && (renderowany tsx dla tego warunku)
)

Reply:
🫡